app/assets/javascripts/discourse/tests/integration/components/user-status-message-test.js

```diff
@@ -126,6 +130,27 @@ module("Integration | Component | user-status-message", function (hooks) {
     );
   });
 
+  test("it accepts optional onTrigger and onUntrigger callbacks", async function (assert) {
```

Comment:
probably don't need all the set here

Reply:
I'm not getting you.`status` is needed and the others are functions for testing. I could unify `onTrigger` and `onUntrigger` into one maybe but I'm not sure you are referring to that.

Reply:
I mean `this.status = ...` instead of `this.set("status", ...)` and so on

Reply:
Oh, ok. Yes, I did it following the other test pattern, but it's repetitive. In that case, it might be best to update all of them